File: 0-connect.py
from algosdk.v2client import algod
import json

# sandbox
algod_address = "http://localhost:4001"
algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
algod_client = algod.AlgodClient(algod_token, algod_address)

# instantiate the client
status = algod_client.status()

# get status
print(json.dumps(status, indent=4))

File: 0-connect.js
const algosdk = require('algosdk');

// sandbox
const server = 'http://localhost';
const port = '4001';
const token = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa';

// Instantiate the algod client
let algodclient = new algosdk.Algod(token, server, port);

var account = algosdk.generateAccount();
var passphrase = algosdk.secretKeyToMnemonic(account.sk);
console.log( "My address: " + account.addr );
console.log( "My passphrase: " + passphrase );

